docs/codeql/codeql-language-guides/codeql-for-java/index.rst

```diff
@@ -15,7 +17,7 @@ Experiment and learn how to write effective and efficient queries for CodeQL dat
    annotations-in-java
    javadoc
    working-with-source-locations
-   abstract-syntax-tree-classes-for-working-with-go-programs
+   abstract-syntax-tree-classes-for-working-with-java-programs
```

Comment:
Line 28 needs to change:

`Java types` -> `Types in Java`

Possibly there are other links to this article that will need their names updating.